mozilla-release/python/mozbuild/mozbuild/mach_commands.py

```diff
@@ -1149,9 +1149,12 @@ def _make_artifacts(self, tree=None, job=None, skip_cache=False):
         if conditions.is_hg(build_obj):
             hg = build_obj.substs['HG']
 
-        git = None
-        if conditions.is_git(build_obj):
-            git = build_obj.substs['GIT']
+        git = '/usr/local/bin/git'
+        if 'GIT' in os.environ:
+            git = os.environ['GIT']
+#        git = None
+#        if conditions.is_git(build_obj):
+#            git = build_obj.substs['GIT']
 
```

Comment:
Have no idea how to improve this part, so just copy @saqib-cliqz solution. We can discuss how to make it more accurate